server/src/main/java/io/druid/server/initialization/jetty/JettyServerModule.java

```diff
@@ -261,8 +278,8 @@ static void initializeServer(Injector injector, Lifecycle lifecycle, final Serve
     try {
       initializer.initialize(server, injector);
     }
-    catch (ConfigurationException e) {
-      throw new ProvisionException(Iterables.getFirst(e.getErrorMessages(), null).getMessage());
+    catch (Exception e) {
```

Comment:
This will swallow interrupted exceptions, is it needed to catch all exceptions here?

Also, is there a reason the ProvisionException wouldn't be able to just take a "caused by"?

Reply:
at this point server startup would fail so it doesn't matter as we would do same for interrupted exception.

Reply:
and, no ProvidsionException does not take caused by.

Reply:
What's special about `ProvisionException`?  Losing the actual stacktrace of an exception generally sucks (especially if this is going to kill the process start).  Can we just wrap into a `RuntimeException` instead?